FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/workmanager/HandlePasswordProtectedMsgWorker.kt

```diff
@@ -135,11 +135,15 @@ class HandlePasswordProtectedMsgWorker(context: Context, params: WorkerParameter
 
             //start of creating and uploading a password-protected msg to FES
             val fromAddress = plainMimeMsgWithAttachments.getFromAddress()
-            val domain = EmailUtil.getDomain(fromAddress.address)
+            val baseFesUrlPath = GeneralUtil.genBaseFesUrlPath(
+              useSharedTenant = !account.useFES,
```

Comment:
Here `account.useFES` still unclear, should be `account.useCustomUrlFes`. After this change, some FES will be used 100% of the time, so `useFES` is no longer a meaningful name.

Reply:
Just for clarity. 

- `CustomUrlFes == true` - means the app should use a customer FES server.
- `CustomUrlFes == false` - means the app should use `shared-tenant-fes`

Am I right?

Reply:
I meant `custom` as in different url, but you interpreted it as `customer` which is actually clearer. So let's call these `useCustomerFesUrl` ? In which case:


 -   useCustomerFesUrl == true - means the app should use a customer FES server.
 -   useCustomerFesUrl == false - means the app should use shared-tenant-fes

Reply:
Perfect. @martgil Please check the current conversation. It has a lot of info about the Android app logic.